internal/sirius/format_date_and_time.go

```diff
@@ -10,6 +10,10 @@ func FormatDateAndTime(formatForDateTime string, dateString string, displayLayou
 		return dateString
 	}
 	stringToDateTime, _ := time.Parse(formatForDateTime, dateString)
-	dateTime := stringToDateTime.Format(displayLayoutDateTime)
+	dateTime := stringToDateTime.Local().Format(displayLayoutDateTime)
 	return dateTime
 }
+
+func isDST() bool {
```

Comment:
What does DST stand for?

Reply:
Also, this function is only used in tests

Reply:
Daylight saving time, do you want it moved into a test suite instead?

Reply:
If it is only used for tests, it should be in the test suite. Is it needed though? It looks like you are only using it to conditionally re-format dates in tests. Can't you instead always re-format the dates in the tests and not bother with the check?

Reply:
I only want to reformat dates half of the year when time.Local (i.e. British Summer Time) doesn't line up with Time.Now (which always uses UTC and is the default in Go). We get a UTC date from Sirius so I do need this reformatting.